content/posts/newsletter-012/index.md

```diff
@@ -66,6 +66,34 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### grr 0.8
+
+[`grr`](https://github.com/msiglreith/grr) is a modern OpenGL 4.5+ wrapper.
+It provides a cleaned up API built around Vulkan's naming scheme.
+The latest release further pushes the crate towards
+full compatibility with the core features.
+
+Most notable changes:
+
+- Added support for a bunch of Formats
+- Extended transfer operations (Attachment <-> Host <-> Buffer <-> Image)
+- Raw context access
+- Shader & Pipeline log control
+- Device submission control
+
+Special thanks to [@masonium](https://github.com/masonium)
+for contributing a lot of these features and fixes!
+
+### ash(-window)
+
+[`ash-window`], an interoperability library for [`ash`] and [`raw-window-handle`],
```

Comment:
Are there any other recent ash updates? Maybe this section could be expanded a little?

Reply:
Unfortunately, there aren't any further news on `ash` right now ):

Reply:
:ok_hand: